test/e2e/specs/editor/blocks/list.spec.js

```diff
@@ -1237,17 +1237,17 @@ test.describe( 'List (@firefox)', () => {
 		await page.keyboard.press( 'Backspace' );
 		await page.keyboard.press( 'Backspace' );
 
-		await expect.poll( editor.getEditedPostContent ).toBe(
-			`<!-- wp:paragraph -->
-<p></p>
-<!-- /wp:paragraph -->
-
-<!-- wp:list -->
-<ul class="wp-block-list"><!-- wp:list-item -->
-<li>2</li>
-<!-- /wp:list-item --></ul>
-<!-- /wp:list -->`
-		);
+		await expect.poll( editor.getBlocks ).toMatchObject( [
```

Comment:
This test seems to have been broken since #42711.

Now it matches the result before the migration:

https://github.com/WordPress/gutenberg/pull/41555/files#diff-97ba09329f315d385933ebd76613f9f2f053ad71d1fb3812cc16b6f7d4fd40aaL83-L87

However, in #42711, it was changed to two paragraphs:

https://github.com/WordPress/gutenberg/pull/42711/files#diff-2d61608781bfdb7c9aea282172506baca36e1c8307f774f3f390328ccee02c67R1066-R1068.

In #45075, it was fixed, but only partially:

https://github.com/WordPress/gutenberg/pull/45075/files#diff-2d61608781bfdb7c9aea282172506baca36e1c8307f774f3f390328ccee02c67L1093-R1097

Since the test has survived for two changes in different PRs, I'm not sure what's the expected behavior anymore 😅. I think the current result makes sense, but I could use some double-checking from past reviewers too. 🙏

Reply:
It seems like the first empty paragraph is actually expected to be there! I've updated the test.